### PR TITLE
Update the README of your repository to include a link to the online version.

### PR DESCRIPTION
**Pull request by VicPeralta**
#Add a link to live demo in Readme file

https://vicperalta.github.io/Portfolio/

Deploy using GitHub Pages

modified:   README.md
modified:   tablet.png

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,7 @@
 <img src="./widescreen.png" width="100%" height="auto"/>
 
 It is still a working project but you can watch a live demo here:  
+
 https://vicperalta.github.io/Portfolio/
```
Comment (brytebee) [inline README.md]: - [x] **Required:** You have added the link as stated, however, you haven't used the appropriate markdown link styling for your link. Example `[Liive Demo](https://live.demo)`.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @VicPeralta 👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- Correctly deployed using Github page✔️
- Live demo works properly without errors✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (VicPeralta) [reply README.md]: Thanks

**Review by Whoistolu — APPROVED** [review]
### Hi @VicPeralta ,

Thank you for implementing the requested changes.

Your project is complete!  There is nothing else to say other than... It's time to merge.

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**